charts/signoz/Chart.lock

```diff
@@ -1,21 +1,6 @@
 dependencies:
 - name: clickhouse
   repository: https://signoz.github.io/charts
-  version: 24.1.2
-- name: k8s-infra
-  repository: https://signoz.github.io/charts
-  version: 0.11.7
-- name: cert-manager
-  repository: https://charts.jetstack.io
-  version: v1.9.1
-- name: ingress-nginx
-  repository: https://kubernetes.github.io/ingress-nginx
-  version: 4.3.0
-- name: minio
-  repository: https://charts.min.io
-  version: 4.0.9
-- name: keycloak
-  repository: https://charts.bitnami.com/bitnami
-  version: 10.1.5
```

Comment:
Nice job cleaning these up. I wanted to get rid of these for sometime now as well.

There will be dependent task of updating docs for keycloak SSO and secure signoz.

Also, with this there will be no infra metrics or logs collection by default with the SigNoz installation.
We will need to update docs accordingly.

Reply:
Can you please point me to the docs? I will update them!

Reply:
- Keycloak - https://signoz.io/docs/tutorial/setting-up-sso-saml-with-keycloak
- Secure SigNoz - https://signoz.io/docs/tutorial/setting-up-tls-for-signoz
- All kubernetes installation docs (EKS/GKE/Others) - https://signoz.io/docs/install/kubernetes